.teamcity/src/main/kotlin/common/performance-test-extensions.kt

```diff
@@ -95,12 +95,15 @@ fun BuildType.cleanUpGitUntrackedFilesAndDirectories() {
     }
 }
 
-fun BuildSteps.cleanUpPerformanceBuildDir(os: Os) {
+fun BuildSteps.cleanUpReadOnlyDir(os: Os) {
     if (os == Os.WINDOWS) {
         script {
-            name = "CLEAN_UP_PERFORMANCE_BUILD_DIR"
+            name = "CLEAN_UP_READ_ONLY_DIR"
             executionMode = BuildStep.ExecutionMode.ALWAYS
-            scriptContent = """rmdir /s /q %teamcity.build.checkoutDir%\subprojects\performance\build && (echo Directory removed) || (echo Directory not found) """
```

Comment:
❓ Why can't we set clean to remove read-only files? I think this should be possible, shouldn't it?

Reply:
Exactly. Doesn't our clean task do that already with some configuration?

Reply:
Looks like it doesn't. Sounds like a feature request we should implement, right?